src/week_2/Part_A_groups.lean

```diff
@@ -311,7 +312,7 @@ begin
   sorry
 end
 
-lemma unique_right_inv {a b : G} (h : a * b = 1) : b = a⁻¹ :=
+lemma unique_right_inv {e : G} (h : ∀ x : G, x * e = x) : e = 1 :=
```

Comment:
That lemma is unique_right_id, right?

Reply:
Oops, indeed! Maybe I'm wrong here then and what was here originally was indeed the lemma that was intended?

The lemma here is "orphan" in that as far as I can tell nothing else uses it (in later files) -- the only reason I thought `unique_right_id` was what was intended was simply because sequentially, the file has you solve left-and-right pairs for `eq_mul_inv/inv_mul`, `mul_<left/right>_eq_self`, `eq_inv/inv_eq_of_mul_eq_one` and then `unique_left_id` (but no right), and here `unique_right_inv` (but no left), but maybe that was simply because it was tiring to do both halves by now? Let me know if I should leave it as is then.